### PR TITLE
Micro.biotech

### PR DESCRIPTION
**Pull request by floradanna**

### CODE REVIEW

**Review by bedroesb — APPROVED** [review]

File: pages/your_domain/microbial_biotechnology.md [hunk 1]
```diff
@@ -7,151 +7,227 @@ related_pages:
...
+#### Build
+The build stage in the microbial biotechnology and/or synthetic biology life cycle is about building of the microbial systems and involves the application of any number of a range of experimental techniques. In this stage the synthetic parts are assembled and transformed into the biological host.
+
+The main aspects of the build stage are:
+* [methods](https://floradanna.github.io/rdm-toolkit/microbial_biotechnology.html#build-methods---documentation-and-metadata-publication), protocols and procedures used to build the modified organism.
+
+#### Test
+The test stage of a biotechnological study is the most variable in terms of the types of data produced. This stage is mostly about:
+* [testing](https://floradanna.github.io/rdm-toolkit/microbial_biotechnology.html#test-outcome-tests---metadata-standards-and-metadata-publication) the outcome or output variables and analyse the modified organism.
+
```
Comment (davidcmarkham) [inline pages/your_domain/microbial_biotechnology.md:33]: Add additional bullet point:
•	Characterising the synthetic parts using experimental data

File: pages/your_domain/microbial_biotechnology.md [hunk 2]
```diff
@@ -7,151 +7,227 @@ related_pages:
...
+#### Learn
+The learning stage consists in interpreting the obtained results, share the acquired knowledge and reuse it in combination with other existing data to improve the creation of the modified organism.
+* Publish and share data and results.
+* Reuse existing data and combine it with new data.
+
```
Comment (davidcmarkham) [inline pages/your_domain/microbial_biotechnology.md:38]: Add additional bullet point:
•	Feed data back into model(s) to inform the next iteration

File: pages/your_domain/microbial_biotechnology.md [hunk 3]
```diff
@@ -7,151 +7,227 @@ related_pages:
...
+* SBOL can also be used to associate a genetic design with its corresponding model.
+* Models can be shared in model repositories such as [biomodels](https://www.ebi.ac.uk/biomodels/).
 
 
-## Build 
+## Build: Methods - Documentation and (meta)data publication
 
-### Description: 
+### Description
+The build stage in the microbial biotechnology and/or synthetic biology life cycle involves the application of any number of a range of experimental techniques and, since these techniques are so varied, the domain is therefore very difficult to standardise in terms of the data and metadata to be shared.
```
Comment (davidcmarkham) [inline pages/your_domain/microbial_biotechnology.md:166]: There is repeated information here from the overall introduction. This is fine if it is intentional as it reminds the reader of the build stage.

File: pages/your_domain/microbial_biotechnology.md [hunk 4]
```diff
@@ -7,151 +7,227 @@ related_pages:
...
 * The database collection from the [International Nucleotide Sequence Database Collaboration](http://www.insdc.org/) provides an umbrella for gathering and sharing a variety of sequence data from different sequence databases internationally.
 * Other databases such as [GenBank](https://www.ncbi.nlm.nih.gov/genbank/) and the [DNA Data Bank of Japan (DDBJ)](https://www.ddbj.nig.ac.jp/index-e.html) also cater for sequence information.
 
 
-### Genetic parts, device and systems
+## Design: Syntethic parts - Existing data, metadata collection and publication
 
-#### Considerations
+### Description
+Approriate and detailed description of the systethic parts design is critical for reproducibility. It is important to consider how to record metadata at each point in the design process in a standard way, so that it can be clear to others and reproducible.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_domain/microbial_biotechnology.md]: done

File: pages/your_domain/microbial_biotechnology.md [hunk 5]
```diff
@@ -7,151 +7,227 @@ related_pages:
...
-* Minimal information standards have been established to specify this metadata, such as the MIGS standard ([Field et al., 2008](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2409278/)).
+* Genomic information can be captured at the nucleotide level using the well-known [European Nucleotide Archive standard (ENA)](https://www.ebi.ac.uk/ena/browser/home) and [submitted](https://www.ebi.ac.uk/ena/browser/submit) to the ENA database to allow the information to be shared.
 * The database collection from the [International Nucleotide Sequence Database Collaboration](http://www.insdc.org/) provides an umbrella for gathering and sharing a variety of sequence data from different sequence databases internationally.
 * Other databases such as [GenBank](https://www.ncbi.nlm.nih.gov/genbank/) and the [DNA Data Bank of Japan (DDBJ)](https://www.ddbj.nig.ac.jp/index-e.html) also cater for sequence information.
 
 
-### Genetic parts, device and systems
+## Design: Syntethic parts - Existing data, metadata collection and publication
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_domain/microbial_biotechnology.md]: done

File: pages/your_domain/microbial_biotechnology.md [hunk 6]
```diff
@@ -7,151 +7,227 @@ related_pages:
...
+#### Metadata and ontologies
 * SBOL allows information about the enzymes and the metabolic pathways to be captured in the design document and so this is a viable approach for sharing more than just the genetics of the system.
-* Databases such as SBOLME ([Kuwahara et al., 2017](https://pubmed.ncbi.nlm.nih.gov/28076956/)) or [SynBioHub](https://synbiohub.org) can be used to share the data. 
-* Metabolite information can also be submitted to, or referred to in, [ChEBI](https://www.ebi.ac.uk/chebi/).
 * Enzymes can be assigned EC numbers, according to the guidance from the [International Union of Biochemistry and Molecular Biology (IUBMB)](https://www.qmul.ac.uk/sbcs/iubmb/), to indicate their function and an entry made in the [BRaunschweig ENzyme DAtabase](https://www.brenda-enzymes.org/) (BRENDA).
 * More generally, the [IUPAC-IUBMB Joint Commission on Biochemical Nomenclature (JCBN)](https://www.qmul.ac.uk/sbcs/iupac/jcbn/) encourages the communication of biochemical information using generally understood terminology.
 
-### Model based designs
+#### (Meta)Data publication
+* SBOL.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_domain/microbial_biotechnology.md]: done

File: pages/your_domain/microbial_biotechnology.md [hunk 7]
```diff
@@ -7,151 +7,227 @@ related_pages:
...
-* The information about how the “final system” was built is highly variable, depending on the DNA synthesis and/or assembly approach used. Consider ways to share this information
+* The information about how the “final system” was built is highly variable, depending on the DNA synthesis and/or assembly approach used. Consider ways to share this information.
 
 ### Solutions
-* Various DNA synthesis companies build DNA from a computer specification of the sequence and also a variety of experimental approaches for assembling DNA molecules. This information can be shared as free text attached to a design in SBOL format and uploaded to a repository that supports SBOL2 format and above such as SynBioHub. 
-
+#### Documentation
 * To the authors’ knowledge, there are no proposed standards that exist that are able to capture this diverse set of data. Currently, from a pragmatic point of view, the best a data manager can do is to make sure data is captured in some form from the lab scientist and grouped together with as much metadata as possible.The metadata standards for a build exercise are still to be defined and so at the discretion of the data manager.
```
[the remark this answers is not in the document]
Comment (floradanna) [reply pages/your_domain/microbial_biotechnology.md]: done

File: pages/your_domain/microbial_biotechnology.md (hunk 1 above)
Comment (floradanna) [reply pages/your_domain/microbial_biotechnology.md:34]: done

File: pages/your_domain/microbial_biotechnology.md (hunk 2 above)
Comment (floradanna) [reply pages/your_domain/microbial_biotechnology.md:40]: done

File: pages/your_domain/microbial_biotechnology.md (hunk 3 above)
Comment (floradanna) [reply pages/your_domain/microbial_biotechnology.md:167]: I think it is intentional indeed.

**Comment by floradanna** [comment]
@davidcmarkham I think I implemented all the changes you suggested. Please, double check.

If you approve the PR, I will merge.

Thank you very much for your corrections and contribution!

**Comment by davidcmarkham** [comment]
> 

Yes, I think that's everything. Happy to merge the changes. Thanks!

**Pull request merged**